### ISSUE TITLE
Home Assistant not as accurate as Merry Sky

### ISSUE THREAD

**Issue by jhemak** [Pirate-Weather/pirateweather#156]
### Describe the bug

Home Assistant and Merry Sky give different current conditions (temperature and state) for the same GPS coordinates. 

### Expected behavior

I would expect the two systems to provide the same or similar results with any differences due to refresh timing. 

### Actual behavior

Merry Sky is providing a more accurate view of current conditions for my location than what is displayed in HA through the Pirate Weather integration. 

### API Endpoint

Production

### Location

GPS Coordinates

### Troubleshooting steps

- [X] I have searched this repository and [Home Assistant Repository](https://github.com/alexander0042/pirate-weather-ha) to see if the issue has already been reported.
- [X] I have read through the [API documentation](https://pirateweather.net/en/latest/API/) before opening this issue.
- [X] I have written an informative title.

**Comment by cloneofghosts** [comment]
What's happening here is Merry Sky caches their forecasts to save on requests so you could be seeing a forecast from hours ago as the current conditions. The differences you are seeing are likely do to the different update intervals between Merry Sky and your Home Assistant integration. I'm also not sure how often Merry Sky refreshes their data nor do I see it stated anywhere on their website.

Basically the HA integration is showing more recent data from the API even though you state it's not as accurate. V2 of the API should hopefully bring some improvements in this area when it's launched sometime this winter.

**Comment by jhemak** [comment]
That is very helpful. Thank you for the explanation!

**Comment by alexander0042** [comment]
Thanks for marking this as closed, and let me know if anything else comes up!
